test/suites/fail-check/002.expected

```diff
@@ -26,3 +26,4 @@ T-002
     ·             ╰── While elaborating
  22 │ }
     ╰────
+  help: The two subterms False and True are not equal.
```

Comment:
I'm wondering if we should use Markdown ticks when referencing terms in error messages? Rust does that.